from_cpython/Include/weakrefobject.h~

```diff
@@ -0,0 +1,83 @@
+/* Weak references objects for Python. */
```

Comment:
haha.  wow I can't believe I did that.  :)

On Thu, Feb 26, 2015 at 4:04 PM, Kevin Modzelewski <notifications@github.com
> wrote:

> In from_cpython/Include/weakrefobject.h~
> <https://github.com/dropbox/pyston/pull/332#discussion_r25478364>:
>
> > @@ -0,0 +1,83 @@
> > +/* Weak references objects for Python. */
>
> Maybe let's add *~ to our .gitignore :P
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dropbox/pyston/pull/332/files#r25478364>.
>